_posts/2017-09-01-sept-01.md

```diff
@@ -0,0 +1,34 @@
+---
+layout: post
+title: "This Week In Erlang Sept 1"
+description: ""
+date: 2017-09-01
+tags: [erlang, news]
+comments: false
+share: true
+---
+
+Hello and welcome to "This week in Erlang"! This week bunch of interesting happened, and most importantly there is a very important announcement! Get excited.
+
+### Community Announcement
+As a community, we want to make the BEAM community better. This requires us to know who we are, what we do, what we like and what we don't like. Today, thanks to *Mariano Guerra*, we are launching the first in what hopefully becomes a yearly survey on the State of the BEAM, please fill the survey and share it widely so we get a good representation of the community.
+
+http://bit.ly/state-of-beam-2017
+
+### Articles and Blog posts
```

Comment:
I found 2 misspelled words in this sentence: 
- Blog
	- *Suggested Replacements:* brog, biog, blo, blot, bloc, bog, slog, blow, bldg, blok, flog, blob, clog, log
- ###